assembler/bsc_assembler.go

```diff
@@ -138,6 +142,22 @@ func (a *BSCAssembler) process(channelId types.ChannelId) error {
 	}
 	logging.Logger.Debugf("start seq and end enq are %d and %d", startSeq, endSequence)
 
+	a.alertSetMutex.Lock()
+	if len(a.alertSet) > 0 {
+		var maxTxSeqOfAlert uint64
+		for k := range a.alertSet {
+			if k > maxTxSeqOfAlert {
+				maxTxSeqOfAlert = k
+			}
+		}
+		if startSeq > maxTxSeqOfAlert {
+			a.metricService.SetHasTxDelay(false)
```

Comment:
what about removing all the items that is before `startSeq`?

Reply:
Actually serving same purpose, as long as there is still alert in the set, keep alerting. Currently Do the celanup once.